packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -123,7 +123,6 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
         <FocusZone>
           <table
```

Comment:
If you're keeping application, it implies its mainly for formatting. If you want it navigable as a true table, it'd need headers and a table description etc